instrumentation/opentelemetry-instrumentation-wsgi/tests/test_wsgi_middleware.py

```diff
@@ -289,7 +289,7 @@ def test_nonstandard_http_method(self):
         self.environ["REQUEST_METHOD"]= "NONSTANDARD"
         app = otel_wsgi.OpenTelemetryMiddleware(simple_wsgi)
         response = app(self.environ, self.start_response)
-        self.validate_response(response, span_name="HTTP UNKNOWN", http_method="UNKNOWN")
+        self.validate_response(response, span_name="UNKNOWN /", http_method="UNKNOWN")
```

Comment:
Where does the / come from?

Reply:
I believe it is what `PATH_INFO` in wsgi environ is defaulting to for this response. Similar to https://github.com/open-telemetry/opentelemetry-python-contrib/blob/main/instrumentation/opentelemetry-instrumentation-wsgi/tests/test_wsgi_middleware.py#L132